gui_qt/litellm_worker.py

```
@@ -0,0 +1,49 @@
+    def run(self) -> None:
+        try:
+            result = LiteLLMSyncBackend(api_key=self.api_key or None).generate(
+                SyncGenerationRequest(
+                    model=self.model,
+                    contents="Reply with OK.",
+                    config={"max_output_tokens": 8, "temperature": 0},
+                )
+            )
+            text = result.response_text.strip()
+            self.completed.emit(True, f"连接成功。模型返回：{text[:80] or '（空响应）'}")
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)